packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -117,6 +137,37 @@ export const GET_ACCOUNT_BY_UID = gql`
         bounceType
         bounceSubType
       }
+      securityEvents {
```

Comment:
Nice 👍🏽, unrelated but I think this would also be useful in a users settings page.